devtools/packaging/scripts/osx/build.sh

```diff
@@ -37,7 +38,7 @@ fi
 mkdir build
 cd build
 cmake ../openmm $CMAKE_FLAGS
-make -j4 all DoxygenApiDocs sphinxpdf
+make -j4 all PythonInstall C++ApiDocs PythonApiDocs sphinxpdf
```

Comment:
> Yes it did. 

Nope. The Doxygen system read the python docstrings from the files `CMAKE_BINARY_DIR`, which did not require that the package be installed.